viewing_party/party.py

```diff
@@ -1,23 +1,226 @@
 # ------------- WAVE 1 --------------------
+import json
+from unittest import result
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Input: movie title, genre, and rating
+    Ouput: If values truthy, produces a dictionary of movies with title, genre, and rating key-value pairs.
+    If inputs are falsy, function returns None.
+    '''
+    movie_dict = {}
+    # if title == True and genre == True and rating == True:
+    if True: 
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+    if title == None or genre == None or rating == None:

```

Comment:
I would recommend moving this conditional to be at the top of your function. This will prevent unnecessary work from being done.